include/SDL3/SDL_init.h

```diff
@@ -39,25 +39,24 @@ extern "C" {
 /* As of version 0.5, SDL is loaded dynamically into the application */
 
 /**
- *  \name SDL_INIT_*
- *
- *  These are the flags which may be passed to SDL_Init().  You should
- *  specify the subsystems which you will be using in your application.
+ * These are the flags which may be passed to SDL_Init().  You should
+ * specify the subsystems which you will be using in your application.
```

Comment:
I briefly looked at other enums and they have a *brief* doc string + a longer one.
I would also add `\sa`, but I don't know whether the sdl wiki supports that on typedef'ed enums.

```suggestion
 *   \brief Initialization flags for SDL_Init and/or SDL_InitSubSystem
 *
 * These are the flags which may be passed to SDL_Init().  You should
 * specify the subsystems which you will be using in your application.
 *
 * \sa SDL_Init
 * \sa SDL_InitSubSystem
```